packages/core/src/amazonq/explorer/amazonQTreeNode.ts

```diff
@@ -60,7 +68,17 @@ export class AmazonQNode implements TreeNode {
         return ''
     }
 
+    public isAmazonQInstalled(): boolean {
+        const extensions = vscode.extensions.all
+        const q = extensions.find(x => x.id === 'amazonwebservices.amazon-q-vscode')
```

Comment:
vscode extension ids are case insensitive. but anyway, vscode already provides `getExtension`, can that be used instead? https://github.com/aws/aws-toolkit-vscode/blob/46560e70b7dd29d83ff557b1db246db5091dd7fe/packages/core/src/shared/utilities/vsCodeUtils.ts#L37

you may also want to check `isActive` : https://github.com/aws/aws-toolkit-vscode/blob/46560e70b7dd29d83ff557b1db246db5091dd7fe/packages/core/src/shared/utilities/vsCodeUtils.ts#L25